src/main/java/com/nike/cerberus/server/config/guice/CmsGuiceModule.java

```diff
@@ -287,4 +298,52 @@ public StaticAssetManager dashboardStaticAssetManager() {
         int maxDepthOfFileTraversal = 2;
         return new StaticAssetManager(DASHBOARD_DIRECTORY_RELATIVE_PATH, maxDepthOfFileTraversal);
     }
+
+    @Provides
+    @Singleton
+    @Named("encryptCryptoMaterialsManager")
+    public CryptoMaterialsManager encryptCryptoMaterialsManager(@Named("cms.encryption.cmk.arns") String cmkArns,
+                                                                @Named("cms.encryption.encrypt.cache.maxSize") int maxSize,
+                                                                @Named("cms.encryption.encrypt.cache.maxAge") int maxAge,
+                                                                @Named("cms.encryption.encrypt.cache.messageUseLimit") int messageUseLimit,
+                                                                Region currentRegion) {
+        MasterKeyProvider<KmsMasterKey> keyProvider = initializeKeyProvider(cmkArns, currentRegion);
```

Comment:
Can you add a log statement here so we can see what the conf is at boot time.

Reply:
Done